forwarder_test.go

```diff
@@ -192,23 +159,23 @@ func TestUpdateResourceIpAddressAndCertificateInfo(t *testing.T) {
 			assert := assert.New(t)
 
 			mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				t.Logf("Received headers: %+v", r.Header)
 				assert.Equal(http.MethodPut, r.Method)
+
 				requestBody, err := io.ReadAll(r.Body)
 				assert.NoError(err)
 				assert.JSONEq(tt.expectedRequestBody, string(requestBody))
 				w.WriteHeader(tt.expectedStatus)
 			}))
-
 			defer mockServer.Close()
 
 			testReq, err := http.NewRequest(http.MethodPut, "/", nil)
 			assert.NoError(err)
 			testReq.Header.Set(realIpHeader, tt.realIP)
 			testReq.Header.Set(certificateProviderHeader, tt.certificateProvider)
-			testReq.Header.Set(expiryDateHeader, tt.expiryDate)
+			testReq.Header.Set(expiryDateHeader, tt.certificateExpiryDate)
 			testReq.Header.Set(deviceCNHeader, tt.deviceCN)
-			testReq.Header.Set("ENVIRONMENT", "test")
-			testReq.Header.Set("X-TENANT-ID", "12345")
```

Comment:
Why have we removed testReq.Header.Set("ENVIRONMENT", "test"), testReq.Header.Set("X-TENANT-ID", "12345")?